config/routes.rb

```diff
@@ -15,6 +15,8 @@
     get "nonprofits"
   end
 
+  resources :student_applications, only: [:new, :create], path: "apply/students"
```

Comment:
I think it's similar to namespace? It just changes the route from
'student_applications/new'
to
'apply/students/new'